deploy/deploy.gradle

```diff
@@ -81,6 +81,43 @@ afterEvaluate { project ->
                         }
                     }
                 }
+
+                pom.withXml {
+                    final dependenciesNode = asNode().appendNode("dependencies")
+                    ext.addDependency = { dep, scope ->
+                        logger.lifecycle("updating dep $dep")
```

Comment:
Can you link to the doc that you used for guidance in comments here?

Reply:
sure, added to the top